vendor/init.bat

```diff
@@ -6,8 +6,49 @@
 :: !!! THIS FILE IS OVERWRITTEN WHEN CMDER IS UPDATED
 :: !!! Use "%CMDER_ROOT%\config\user-profile.cmd" to add your own startup commands
 
-:: Set to > 0 for verbose output to aid in debugging.
-if not defined verbose-output ( set verbose-output=0 )
+:: Use -v command line arg or set to > 0 for verbose output to aid in debugging.
```

Comment:
Either change `-v` to `/v`, or make `init.bat` accept `-v` and `--git-install-root` formatted arguments as well as `/v` and `/git_install_root` format.

Personally I vote for the latter.

Reply:
It is dos not linux/unix sticking with / cmder.exe uses /.  in batch to support - and -- and / is a lot of duplicated code.  fixed the mention of -v to be /v